Source/WebCore/platform/graphics/MediaPlayer.cpp

```diff
@@ -2071,6 +2074,17 @@ const Logger& MediaPlayer::mediaPlayerLogger()
 }
 #endif
 
+void MediaPlayer::setMessageClientForTesting(WeakPtr<MessageClientForTesting> internalMessageClient)
+{
+    m_internalMessageClient = internalMessageClient;
```

Comment:
```suggestion
    m_internalMessageClient = WTFMove(internalMessageClient);
```

Reply:
Can't do that without breaking the line below. But I'll re-jigger things.